advanced/end/app/src/main/java/com/example/android/codelabs/paging/data/GithubRemoteMediator.kt

```diff
@@ -82,8 +81,7 @@ class GithubRemoteMediator(
             }
         }
 
-        val apiQuery = query + IN_QUALIFIER
-
+        val apiQuery = query
```

Comment:
Why is the query we pass to Github being changed?

Reply:
I suspect the github api changed at one point because the existing IN_QUALIFIER syntax no longer works (0 results returned). Right now, the apiQuery alone is already returning the results it used to (repositories containing Android content and sorted from most popular to least).